### PR TITLE
SQLStatement: getParams and getParameters are redundant

### PR DESCRIPTION
**Pull request by stevenschlansker**

### CODE REVIEW

File: core/src/main/java/org/jdbi/v3/Query.java [hunk 1]
```diff
@@ -114,7 +114,7 @@
      */
     public <T> Query<T> mapTo(Class<T> resultType)
     {
-        return this.mapTo((Type) resultType);
+        return this.mapTo(resultType);
```
Comment (qualidafial) [inline core/src/main/java/org/jdbi/v3/Query.java]: StackOverflow coming from this change.

Comment (stevenschlansker) [inline core/src/main/java/org/jdbi/v3/Query.java]: Crap!  Stupid "helpful" IDE!!!

**Comment by qualidafial** [comment]
Any reason this PR is merging into `jdbi3-pg` branch instead of `jdbi3`?

**Comment by stevenschlansker** [comment]
Crap, because I'm bad at Git :(

**Pull request closed without merging**